test/fixtures/ast/v-bind-same-name-shorthand07-camelcase/scope.json

```diff
@@ -0,0 +1,154 @@
+{
+    "type": "global",
+    "variables": [],
+    "references": [],
+    "childScopes": [
+        {
+            "type": "module",
+            "variables": [
+                {
+                    "name": "src",
```

Comment:
Could you please run `npm run update-fixtures`?

Reply:
done ✅